pkg/sshagent/sshagent.go

```diff
@@ -111,8 +111,9 @@ func (a *AgentServer) Serve(processLabel string) (string, error) {
 				a.wg.Done()
 			}()
 			// the only way to get agent.ServeAgent is to close the connection it's serving on
+			// TODO: ideally we should use some sort of forwarding mechanism for output instead of manually closing connection.
 			go func() {
-				time.Sleep(500 * time.Millisecond)
+				time.Sleep(2000 * time.Millisecond)
```

Comment:
I'm OK with the value, but just wonder if we could instead get this value from a config file.

Reply:
@TomSweeneyRedHat Sure we could that but I am confused if this should be visible to end-user and anybody would ever want to configure this.

Ideally we should remove timeout and add buffer forwarding mechanism which should close connection but that seemed like a bigger change to me and would like to do refactor bunch of things in `sshagent` after discussing with maintainers in a separate thread.

Reply:
No to config, since users will have no idea what to set it.  None of us have any idea either, since it will probably vary under load.

I say increase the hack for now, and then consider fixing this long term in the future.
LGTM

Reply:
I concur with Dan.  At least I would have absolutely no feeling for what a good value for that would be, and also be worried why I have to make that decision.

Reply:
FWIW, it would not be a config setting I'd heavily document.  More of a tool for support if the issue popped up in the future, or we found out that 2,000 was too much and was causing some other unanticipated error.  I think Dan and Valentin being unsure about a valid value is why I would like to make it configurable.  But hopefully, this will fix this, and it will be a non-issue.

Reply:
I would say that if this happens again in the field, then we should put in the work to build a complete solution.